packages/angular/package.json

```diff
@@ -1,6 +1,6 @@
 {
```

Comment:
Regarding SDK development, do we check if the SDK/examples work with different Node versions?